app/templates/application.hbs

```diff
@@ -239,6 +267,8 @@
         {{#menu.menu-item
           title=layerGroup.model.title
           tooltip=layerGroup.model.titleTooltip
+          legendColor=layerGroup.model.legendColor
+          legendIcon=layerGroup.model.legendIcon
```

Comment:
I'm slightly wary of this only because it's looking like menu-item should just require a simple layer-group `model`. Then the API will be simpler. Could be a possible refactoring.